client/components/editor/teaching-elements/Assessment/TextSelection/helpers.js

```diff
@@ -0,0 +1,128 @@
+import Blast from 'blast-vanilla';
+import first from 'lodash/first';
+import generateRange from 'lodash/range';
```

Comment:
Avoid this because later on it is almost impossible to guess from code that you are actually using lodash's utility.